python/lsst/ts/lasertracker/mock/mock_utils.py

```diff
@@ -38,7 +38,7 @@
     m1m3=MockT2SAPointGroup(
         origin=CartesianCoordinate(0.0, 0.0, 0.0),
         rotation=BodyRotation(0.0, 0.0, 0.0),
-        radius=8.40,
```

Comment:
Why are you changing these?

Reply:
Because 8.40 is the diameter of the telescope, not the radius, no?